### PR TITLE
MB-15874: show edit/review requests link, yellow alert and yellow tag for SIT address change request

### PR DESCRIPTION
**Pull request by StuTruss**
## Summary

This PR implements the designs shown [here](https://www.figma.com/file/5AhuEkaH3bJVrtffU3wp22/Milmove-Office-App?node-id=10509-115679&t=he6qPm8lP6yYnK7c-4)

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers
- [x] Link In Approved service item section: 
  - [x] "Edit" link if there are no SIT address changes
  - [x] "Review request" link if there are SIT address change requests
 - [x] Yellow notification on MTO page: "Service item update requested. Review request below."
 - [x] Yellow status label on MTO page Approved Service Item section: "UPDATE REQUESTED"
 - [ ] Has unit test

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Access `http://officelocal:3000/testharness/list` and scroll down to button that says `HHGMoveInSITWithAddressChangeRequestOver50Miles` then click it. 
2. Copy value of `locator` field
3. Access the Office App
4. Login as a TOO
5. Navigate to move with value of `locator` field you copied earlier
6. Click `Move task order` tab

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/about/supported-browsers) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

## Screenshots
Yellow notification saying "Service Item update requested. Review request below"
Screenshot 2023-05-17 at 2 48 57 PM

Yellow status label saying "UPDATE REQUESTED"
Screenshot 2023-05-17 at 2 49 12 PM

### CODE REVIEW

**Review by afoon — CHANGES_REQUESTED** [review]
LGTM. The unit test for `MoveTaskOrder.jsx` needs to be updated to test for the new alert and new variable that can affect notification count. `ServiceItemTable.jsx` needs a unit test for the new Edit button.

File: src/shared/constants.js [hunk 1]
```diff
@@ -100,6 +100,12 @@ export const SERVICE_ITEM_STATUS = {
   REJECTED: 'REJECTED',
 };
 
+export const SIT_ADDRESS_UPDATE_STATUS = {
```
[the remark this answers is not in the document]
Comment (michaelvaldes-truss) [reply src/shared/constants.js:103]: **suggestion:** constants/sitUpdates.js might also be a good place to store information about which service items can have address updates in the first place (DDDSIT, which is defined in src/content/serviceItems.js)

File: src/components/Office/ServiceItemsTable/ServiceItemsTable.jsx [hunk 2]
```diff
@@ -35,84 +35,98 @@ const ServiceItemsTable = ({
...
+                    <span className="icon">
+                      <FontAwesomeIcon icon="times" />
+                    </span>{' '}
+                    Reject
+                  </Button>
+                  <Button type="button" className="text-blue usa-button--unstyled margin-left-1">
+                    <span>
+                      <FontAwesomeIcon icon="pencil" style={{ marginRight: '5px' }} />
+                    </span>{' '}
+                    {sitAddressUpdates && hasSITAddressUpdate(sitAddressUpdates) ? 'Review Request' : 'Edit'}
```
Comment (michaelvaldes-truss) [inline src/components/Office/ServiceItemsTable/ServiceItemsTable.jsx:103]: **suggestion:** as @afoon pointed out on slack, we only want this button to appear for DDDSIT service items

**Review by michaelvaldes-truss — CHANGES_REQUESTED** [review]
Left a few suggestions, but this is looking really great so far!
One more thing to note: pending @AnnaGingle 's judgement, we might also need to increment the "move task order" tab tag at the top, similar to how we're incrementing the shipment tag on the left in this PR.

**Review by AnnaGingle — CHANGES_REQUESTED** [review]
- Only the Domestic destination SIT delivery service item should have an Edit/Review button
- MTO top nav should have a Yellow tag like the MTO sidenav
- I need this test move to have risk of excess weight so I can confirm the alerts play nice.
- (not apart of this work but documenting because i don't see it in other jira stories). The move should have a 'Approvals requested' status. Right now it shows move approve. TOOs work on a first in first out basis. They would never think to click on this move otherwise.

**Comment by StuTruss** [comment]
@AnnaGingle this is currently how the two alerts look on the page when they are both shown together.
Screenshot 2023-05-18 at 5 20 20 PM

**Comment by AnnaGingle** [comment]
> @AnnaGingle this is currently how the two alerts look on the page when they are both shown together. Screenshot 2023-05-18 at 5 20 20 PM
The new alert should match the excess weight alert. Not sure why they are different sizes.

File: src/pages/Office/MoveTaskOrder/MoveTaskOrder.jsx [hunk 3]
```diff
@@ -716,6 +754,11 @@ export const MoveTaskOrder = (props) => {
               Your changes were saved
             </Alert>
           )}
+          {renderSITAddressUpdateAlert && (
+            <Alert type="warning" headingLevel="h4" className={styles.alertWithButton}>
```
Comment (afoon) [inline src/pages/Office/MoveTaskOrder/MoveTaskOrder.jsx:758]:
```suggestion
            <Alert type="warning"  className={styles.alertWithButton}>
```
removing the heading level should fix the sizing.

**Review by afoon — APPROVED** [review]
Outside of the design changes requested, this LGTM.

**Comment by StuTruss** [comment]
@AnnaGingle Made the alert slim and added a close btn to it
Screenshot 2023-05-19 at 1 29 35 PM

**Review by michaelvaldes-truss — APPROVED** [review]
Lgtm! Thanks for all your work on this, there was a lot of minutia to handle 🥇

**Review by AnnaGingle — CHANGES_REQUESTED** [review]
- The top nav MTO tag is not working as expected ([link to video](https://ustcdp3.slack.com/archives/CTNTFJSBA/p1684529911844029?thread_ts=1684527876.902239&cid=CTNTFJSBA))
- The service item tag should be ABOVE the service item title ([link to picture & designs](https://ustcdp3.sla
Screenshot 2023-05-19 at 3 59 46 PM
ck.com/archives/CTNTFJSBA/p1684530170684859?thr

Uploading Screen Recording 2023-05-19 at 3.56.43 PM.mov…

ead_ts=1684527876.902239&cid=CTNTFJSBA))

**Review by AnnaGingle — APPROVED** [review]
Great work! Approved!

**Pull request merged**